DOC: Get rid of question marks in getting started tutorials

samrao1997 opened the pull request:
- [x] closes #48227 (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

samrao1997 committed:
Get rid of question marks in getting started tutorials

mroeschke commented on doc/source/getting_started/intro_tutorials/04_plotting.rst:
```diff
@@ -2,7 +2,7 @@
 
-How do I create plots in pandas?
+How to create plots in pandas
 ----------------------------------
```
All the `-` will need to match the length of these altered titles.

MarcoGorelli reviewed (changes requested):
The issue is about removing question marks when the heading isn't a question (e.g. "How to calculate summary statistics?"), heading that are already questions (e.g. "How do I create plots in pandas?") can stay as they are

samrao1997 committed:
get rid of unneeded change

samrao1997 commented:
@MarcoGorelli I adjusted that to remain as is [aka `How do I create plots?`]. 

However, I noticed that in the docs for some reason the [plots](https://pandas.pydata.org/docs/getting_started/intro_tutorials/04_plotting.html#) heading shows `How to` with a question mark and when I added the previous change with `How to` without question mark building the docs got rid of the question mark on that page.

Not sure how the code show `How do I` when the docs show the heading `How to`....

MarcoGorelli commented on doc/source/getting_started/intro_tutorials/06_calculate_statistics.rst:
```diff
@@ -30,7 +30,7 @@
     </div>
 
-How to calculate summary statistics?
+How to calculate summary statistics
 ------------------------------------
```
you need to align the underscores (remove the last one)

MarcoGorelli commented:
> Not sure how the code show `How do I` when the docs show the heading `How to`....

you probably just need to do `python make.py clean` first

samrao1997 committed:
Merge branch 'main' into get-rid-of-questmark-tutorials

samrao1997 committed:
Realign dashes with length of altered titles

samrao1997 commented:
@MarcoGorelli I have updated the dashes to be the length of the alter titles. Sorry about that.

MarcoGorelli reviewed (approved):
Thanks @samrao1997 

![](https://i.fluffy.cc/qHzfcB6BZ9ZD7K3wFknqLsrHqsq56bPM.gif)

mroeschke commented:
Thanks @samrao1997
